SU2_CFD/src/numerics_direct_mean.cpp

```diff
@@ -4413,10 +4413,18 @@ void CAvgGrad_Flow::ComputeResidual(su2double *val_residual, su2double **val_Jac
     }
   }
   
+  /*--- Wall shear stress values (wall functions) ---*/
+  
+  if (TauWall_i > 0.0 && TauWall_j > 0.0) Mean_TauWall = 0.5*(TauWall_i + TauWall_j);
+  else if (TauWall_i > 0.0) Mean_TauWall = TauWall_i;
+  else if (TauWall_j > 0.0) Mean_TauWall = TauWall_j;
+  else Mean_TauWall = -1.0;
```

Comment:
One concern that I had in my original implementation was that this check for positive/negative numbers was not very elegant... don't remember what my original motivation was.. perhaps we should just use a boolean?